test/deprecations.js

```diff
@@ -0,0 +1,50 @@
+var morgan = require('..')
+var assert = require('assert')
+
+describe('morgan() deprecations', function () {
+  var originalConsoleError
+  var errors = []
+  function getErrors () {
+    return errors.map(function (e) { return removeANSI(e.toString('utf8')) })
+  }
+  before(function () {
+    originalConsoleError = process.stderr.write
+
+    process.stderr.write = function write (chunk, encoding) {
+      /* eslint node/no-deprecated-api: 0 */ // this is actually trying to be compat with older node
```

Comment:
Rather than disabling it, I would follow the suggestion of the linter and use `safe-buffer` instead.

Reply:
Ok, I thought you wouldn't like to add a new package just for this.

Reply:
It's just a devDependency since this is a test.